front-end/src/config.js

```diff
@@ -0,0 +1,5 @@
+const port = process.env.MATADOR_PORT || 3000;
```

Comment:
Now we have the default value defined in two places... :thinking: 

Might seem overkill, but depending on how much we plan on extending this, I suggest creating a single `env.js` that the config files can import (or just have a single `config.js` file). 

There's also the convention to create an `.env` file, but it's no better or worse than the current alternative (at least not for our purposes) :+1:

Reply:
Hmm, I started out with one file but was not allowed to import files from outside of the src folder, since in a way they are different projects. We might want to define them in another way in the future, maybe as part of a script?